resources/hpxml-measures/BuildResidentialHPXML/measure.rb

```diff
@@ -123,6 +123,11 @@ def arguments(model)
     arg.setDefaultValue('USA_CO_Denver.Intl.AP.725650_TMY3.epw')
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument.makeStringArgument('fips_code', false)
+    arg.setDisplayName('County FIPS Code')
+    arg.setDescription('County FIPS Code - used to calculate Home Energy Scores from ResStock-generated files')
+    args << arg
```

Comment:
Similar to vintage comment, change this argument to `zip_code` and use ResStockArguments measure to translate `fips_code`.